controller/login_test.go

```diff
@@ -148,6 +162,68 @@ func (rest *TestLoginREST) TestLinkIdPWithTokenRedirects() {
 	test.LinkLoginTemporaryRedirect(t, svc.Context, svc, ctrl, nil, nil)
 }
 
+func (rest *TestLoginREST) TestResourceRequestPayload() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
```

Comment:
We aren't actually making a remote KC call here - all we are doing is validating the conversion from `recentSpaces` to the `resource` payload.